msg/en_US/csql.msg

```diff
@@ -176,7 +178,7 @@ $ csql help messages
    ;SChema class-name           - display schema information of a class.\n\
    ;TRIgger [`*'|trigger-name]  - display trigger definition.\n\
    ;Get system_parameter        - get the value of a system parameter.\n\
-   ;SEt system_parameter=value  - set the value of a system parameter.\n\
+   ;SEt system_parameter=value  - set the value of a system parameter.(DBA only)\n\
```

Comment:
try following with 'public'
`csql> ;set optimization_level 513`

So, I think following is proper:
`   ;SEt system_parameter=value  - set the value of a system parameter.`

Reply:
Oh... does manual have typo? Ok I will apply it.